tests/ParallelRestoreOldBackupApiCorrectnessAtomicRestore.toml

```diff
@@ -1,3 +1,7 @@
+[configuration]
+# FIXME: re-enable rocks at some point
+storageEngineExcludeTypes = [4]
```

Comment:
Backup tests are disabled in SimulatedCluster. why are we doing it here?

Reply:
Reverted.